app/component/SummaryRow.js

```diff
@@ -143,7 +151,7 @@ export default function SummaryRow(props, { breakpoint, intl: { formatMessage }
         <div
           className={cx('itinerary-start-time', { 'realtime-available': realTimeAvailable })}
           key="startTime"
-        >
+        ><span className={cx('itinerary-start-date', { nobg: sameDay })} ><span>{dateOrEmpty(startTime, NOW)}</span></span>
```

Comment:
This should also compare the requested date, not today's date. Also consider splitting the elements to individual rows for readability